api/v1/model_types.go

```diff
@@ -144,6 +148,33 @@ type Adapter struct {
 	URL string `json:"url"`
 }
 
+type LoadBalancing struct {
+	Strategy LoadBalancingStrategy `json:"strategy"`
+	CHWBL    CHWBL                 `json:"chwbl"`
+}
+
+// +kubebuilder:validation:Enum=LeastLoad;CHWBL
+type LoadBalancingStrategy string
+
+const (
+	LeastLoadStrategy LoadBalancingStrategy = "LeastLoad"
+	CHWBLStrategy     LoadBalancingStrategy = "CHWBL"
```

Comment:
nit, prefer to use the full name instead of abbreviations that most people don't understand. It's fine if it's long.